client/package.json

```diff
@@ -26,6 +27,8 @@
     "start": "react-scripts start",
     "prebuild": "ts-node -O {\\\"module\\\":\\\"commonjs\\\"} tools/generate-sitemap.ts",
     "build": "react-scripts build",
+    "predeploy": "npm run build",
+    "deploy": "gh-pages -d build",
```

Comment:
Small: Chuck these two down one line - just to keep the `prebuild`, `build` and `postbuild` grouped.